main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -1949,8 +1917,6 @@ Outputs:
     Value: !GetAtt RoleWorkflowLoopRunner.Arn
   OpenDataScrapeHandlerRoleArn:
     Value: !GetAtt RoleOpenDataScrapeHandler.Arn
-  EnvStatusPollHandlerRoleArn:
-    Value: !GetAtt RoleEnvStatusPollHandler.Arn
```

Comment:
Do we need to update this configuration as well: https://github.com/awslabs/service-workbench-on-aws/blob/4b5928ebbbd837d2e20781a213c2fcb630fbdd95/main/solution/backend/serverless.yml#L73-L75

Reply:
Good catch!